### PR TITLE
Config git hooks

### PR DESCRIPTION
**Pull request by KarenVentura**
- Install Rubocop gem
- Install overcommit gem for git hooks
- add pre-commit hook hook
- add commit-msg
- add pre-push hook

### CODE REVIEW

File: .overcommit.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    enabled: true
+  CoffeeLint:
+    enabled: true
+  ScssLint:
+    enabled: true
+  HamlLint:
+    enabled: true
+  RubyLint:
+    enabled: true
+  TravisLint:
```
Comment (mumoc) [inline .overcommit.yml]: What does travis lint do?

**Comment by mumoc** [comment]
LGTM, merge merge

File: .overcommit.yml (hunk 1 above)
Comment (KarenVentura) [inline .overcommit.yml]: TravisLint analize the travis files against any modified

Comment (mumoc) [inline .overcommit.yml]: oh, ok

**Pull request merged**